libclc/x86_64-unknown-linux/libspirv/math/native_exp.cl

```diff
@@ -0,0 +1,4 @@
+#define IS_NATIVE 
+#include "helpers.h"
+
+GEN_UNARY_BUILTIN(exp)
```

Comment:
I think `IS_NATIVE` could be staying defined when you don't want it to?

```suggestion
GEN_UNARY_BUILTIN(exp)
#undef IS_NATIVE
```

etc

Reply:
Good point, that's done, thank you